src/snowflake/snowpark/modin/plugin/_internal/timestamp_utils.py

```diff
@@ -443,43 +443,35 @@ def raise_if_to_datetime_not_supported(
 
 
 def convert_dateoffset_to_interval(
-    value: native_pd.DateOffset,
+    value: native_pd.DateOffset, _emit_ast: bool = True
 ) -> Interval:
     """
     Converts a pandas DateOffset where value is treated as a timedelta to a Snowpark
     Interval keyword. DateOffset with parameters that replace the offset value is not
     yet supported, so a NotImplemented error is raised.
     """
```

Comment:
Was talking with @sfc-gh-nkrishna about this here, after some discussion concluded to add comment to clarify what's missing in pandas.